TheJumpCloud/support
# Sa 3140 set new policy functions (#483)

## Issues
* [SA-3140](https://jumpcloud.atlassian.net/browse/SA-3140) - Release New/Set-JCPolicy

## What does this solve?

Release 1 of New/Set-JCpolicy Functions. PR #483 was already approved with tests and changes made, this PR is the feature branch merge to main branch. 

## Is there anything particularly tricky?

See #483 

## How should this be tested?

See #483 

[SA-3140]: https://jumpcloud.atlassian.net/browse/SA-3140?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
